CHANGELOG.rst

```diff
@@ -7,10 +7,10 @@ xskillscore v0.0.16 (2020-XX-XX)
 
 Internal Changes
 ----------------
-
 - Add community support documents: ``HOWTOCONTRIBUTE.rst``, issue template and pull request template. `Aaron Spring`_
-- Replace pandas with cftime in examples and tests. `Aaron Spring`_ 
-- Add coveralls. `Aaron Spring`_ 
+- Replace ``pandas`` with ``cftime`` in examples and tests. `Aaron Spring`_
+- Add coveralls for tests coverage. `Aaron Spring`_
+- Add ``pre-commit`` for formatting. `Aaron Spring`_
```

Comment:
Add also yourself for these PRs

Reply:
will do. your name first as you did the leg work and I just copied and pasted :)